.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: go
+
+os:
+  - linux
+
+go:
+- 1.11.x
+
+git:
+  depth: 1
+
+# Dependencies included in vendor
+install: true
+
+script:
+  - go test
```

Comment:
does this work with dependencies?  I.E. if a unit test requires a dependency does this work

Reply:
The dependencies are vendored into the repo so this should work (set up Travis with this same configuration on my fork).

Reply:
I'm only familiar with godep.  With that, I need to run `godep go test` rather than `go test`.  But if this works for dep, than this is good to go